lib/util.js

```diff
@@ -125,6 +125,11 @@ util.isArrayBufferView = function(x) {
   return x && util.isArrayBuffer(x.buffer) && x.byteLength !== undefined;
 };
 
+// Node versions >= 10.12.0 support native keyPair generation,
+// which is non-deterministic
+util.isDeterministic = util.isNodejs &&
+  typeof require('crypto').generateKeyPair !== 'function';
```

Comment:
It seems odd to export this as part of the main API.  Maybe the name is just too generic?  Is this something people would use outside of testing?  If not it could just be in the rsa test file.

Reply:
Yeah, that's fair. Will move to the test file.